packages/ag-charts-community/src/api/preset/sparkline.ts

```diff
@@ -235,15 +242,27 @@ function axisPreset(
     return { type: defaultType };
 }
 
-function gridLinePreset(opts: AgSparklineAxisOptions | undefined, defaultEnabled: boolean): AgAxisGridLineOptions {
+function gridLinePreset(
+    opts: AgSparklineAxisOptions | undefined,
+    defaultEnabled: boolean,
+    sparkOpts: AgSparklineOptions
+): AgAxisGridLineOptions {
     const gridLineOpts: AgAxisGridLineOptions = {};
+    const isBarVertical = sparkOpts.type === 'bar' && sparkOpts.direction !== 'horizontal';
+
     if (opts?.stroke != null) {
         gridLineOpts.style = [{ stroke: opts?.stroke }];
         gridLineOpts.enabled ??= true;
+    } else if (isBarVertical) {
+        gridLineOpts.style = [{ stroke: DEFAULT_AXIS_GRID_COLOUR }];
+        gridLineOpts.enabled ??= true;
     }
     if (opts?.strokeWidth != null) {
         gridLineOpts.width = opts?.strokeWidth;
         gridLineOpts.enabled ??= true;
+    } else if (isBarVertical) {
+        gridLineOpts.width = 2;
```

Comment:
We should avoid hardcoding where possible, why do we need a vertical-case specific hardcoding too? I'd normally suggest where feasible to put this fallback default in the theme.

Reply:
It's only applicable for vertical bar charts, and our themes don't allow that level of specificity. But, as the default for horizontal bar charts is hidden anyway, we can move this default to the theme

Reply:
Moved to theme